org-code-javabuilder/playground/src/main/java/org/code/playground/ClickableItem.java

```diff
@@ -0,0 +1,20 @@
+package org.code.playground;
+
+import java.io.FileNotFoundException;
+
+public abstract class ClickableItem extends Item {
+
+    /**
+     * Sets a sound to play when the user clicks on this item.
+     * 
+     * @param filename the name of the sound file from the asset manager to play
+     * @throws FileNotFoundException when the sound file cannot be found.
+     */
+    public void setClickSound(String filename) throws FileNotFoundException;
```

Comment:
To confirm: We don't want to allow students to build their own sounds, correct? Just play pre-made sounds? Would we ever want to allow student-built sounds?

Reply:
No, I think for the same reasons we won’t allow image manipulation - performance and ensuring no confusion between this and the theater API.

Reply:
👍